src/views/lists_home.js

```diff
@@ -131,7 +138,9 @@ class Icons extends Component {
         key={sectionID}
         onPress={log}
         title={rowData.title}
-        icon={{ name: rowData.icon }}
+        leftIcon={{ name: rowData.icon }}
+        disclosure
```

Comment:
is this what we are naming it? `disclosure`

Is that the arrow on the right? if I am confused.. chances are other will be too. can we change it?

Reply:
I took it from iOS, chevron is more like the character name ">", I think the real name in design is disclosure. But it can be strange to old user you're right. So back to chevron ?

Reply:
Yeah, let's make is `chevron`, it's def more clear and I have seen multiple people use it before. 
Sorry for the changes. 😢